testnet-contracts/contracts/Bank.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity ^0.5.0;
+
+import "../../node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./BankToken.sol";
+
+/**
+ * @title Bank
+ * @dev Manages the deployment and minting of ERC20 compatible BankTokens
+ **/
+
+contract Bank {
```

Comment:
Thinking about naming, maybe should be BridgeBank?

Reply:
Renamed this to CosmosBank in https://github.com/cosmos/peggy/pull/76/commits/89cb10e8d06ff3dcd467bcfc7b2821bf14240153 (with Processor renamed to EthereumBank for clarity). Let's discuss an architecture along the lines of: EthereumBank, CosmosBank -> BridgeBank -> Peggy.